test/groovy/PiperStageWrapperTest.groovy

```diff
@@ -181,6 +185,25 @@ class PiperStageWrapperTest extends BasePiperTest {
         assertThat(DebugReport.instance.localExtensions.test_old_extension, is('Extends'))
     }
 
+    @Test
+    void testExtensionExec() {
+        helper.registerAllowedMethod('fileExists', [String.class], { path ->
+            return (path == '.pipeline/extensions/test_old_extension.groovy')
+        })
+
+        nullScript.commonPipelineEnvironment.gitBranch = 'testBranch'
+        nullScript.env.PIPER_DISABLE_EXTENSIONS = true
+
+        stepRule.step.piperStageWrapper(
+            script: nullScript,
+            juStabUtils: utils,
+            ordinal: 10,
+            stageName: 'test_old_extension'
+        ) {}
+        //setting above parameter to 'true' bypasses the below message
+        assert !loggingRule.log.contains('[piperStageWrapper] Running project interceptor \'.pipeline/extensions/test_old_extension.groovy\' for test_old_extension.')
```

Comment:
Using Hamcrest Matchers ([Quick Reference](https://www.marcphilipp.de/downloads/posts/2013-01-02-hamcrest-quick-reference/Hamcrest-1.3.pdf)) the code is a bit easier to read. 
```suggestion
        assertThat(loggingRule.log, not(containsString("[piperStageWrapper] Running project interceptor '.pipeline/extensions/test_old_extension.groovy' for test_old_extension."))
```

Reply:
I used this initially, but for some reason it throws this exception :
```
groovy.lang.MissingMethodException: No signature of method: PiperStageWrapperTest.not() is applicable for argument types: (org.hamcrest.core.StringContains) values: [a string containing "[piperStageWrapper] Running project interceptor '.pipeline/extensions/test_old_extension.groovy' for test_old_extension."]
Possible solutions: init(), notify(), wait(), any(), wait(long), is(java.lang.Object)
```

Reply:
Yes, you need to do a static import.